modules/deploy/partials/cloud/tiers.adoc

```diff
@@ -0,0 +1,610 @@
+
+|=== 
+| Tier | Maximum ingress | Maximum egress | Maximum logical partitions | Maximum connections
+
+| Tier 1 | 20 MBps | 60 MBps | 1,000 | 1,500
+| Tier 2 | 50 MBps | 150 MBps | 2,800 | 3,750
+| Tier 3 | 100 MBps | 200 MBps | 5,600 | 7,500
+| Tier 4 | 200 MBps | 400 MBps | 11,300 | 15,000
+| Tier 5 | 400 MBps | 800 MBps | 22,800 | 30,000
+|===
+
+NOTE: Serverless clusters are not on tiers. See xref:deploy:deployment-option/cloud/serverless.adoc#limits[Serverless limits]. To change your cluster type, contact https://support.redpanda.com/hc/en-us/requests/new[support^].
+
+== Dedicated supported regions
+
+[tabs]
+====
+Google Cloud Platform (GCP)::
++
+--
+|=== 
+| Region | Zones | Available throughput tiers
+| asia-south1 
+a| * asia-south1-a 
+* asia-south1-b 
+* asia-south1-c 
+a| * tier-1-gcp-v2-x86
+* tier-2-gcp-v2-x86
+* tier-3-gcp-v2-x86
+* tier-4-gcp-v2-x86
+* tier-5-gcp-v2-x86
+| asia-southeast1
+a| * asia-southeast1-a
+* asia-southeast1-b
+* asia-southeast1-c
+a| * tier-1-gcp-v2-x86
+* tier-2-gcp-v2-x86
+* tier-3-gcp-v2-x86
+* tier-4-gcp-v2-x86
+* tier-5-gcp-v2-x86
+| australia-southeast1
+a| * australia-southeast1-a
+* australia-southeast1-b
+* australia-southeast1-c
+a| * tier-1-gcp-v2-x86
+* tier-2-gcp-v2-x86
+* tier-3-gcp-v2-x86
+* tier-4-gcp-v2-x86
+* tier-5-gcp-v2-x86
+| europe-west1
+a| * europe-west1-b
+* europe-west1-c
+* europe-west1-d
+a| * tier-1-gcp-v2-x86
+* tier-2-gcp-v2-x86
+* tier-3-gcp-v2-x86
+* tier-4-gcp-v2-x86
+* tier-5-gcp-v2-x86
+| europe-west2
+a| * europe-west2-a
+* europe-west2-b
+* europe-west2-c
+a| * tier-1-gcp-v2-x86
+* tier-2-gcp-v2-x86
+* tier-3-gcp-v2-x86
+* tier-4-gcp-v2-x86
+* tier-5-gcp-v2-x86
+| europe-west3
+a| * europe-west3-a
+* europe-west3-b
+* europe-west3-c
+a| * tier-1-gcp-v2-x86
+* tier-2-gcp-v2-x86
+* tier-3-gcp-v2-x86
+* tier-4-gcp-v2-x86
+* tier-5-gcp-v2-x86
+| northamerica-northeast1
+a| * northamerica-northeast1-a
+* northamerica-northeast1-b
+* northamerica-northeast1-c
+a| * tier-1-gcp-v2-x86
+* tier-2-gcp-v2-x86
+* tier-3-gcp-v2-x86
+* tier-4-gcp-v2-x86
+* tier-5-gcp-v2-x86
+| us-central1
+a| * us-central1-a
+* us-central1-b
+* us-central1-c
+* us-central1-f
+a| * tier-1-gcp-v2-x86
+* tier-2-gcp-v2-x86
+* tier-3-gcp-v2-x86
+* tier-4-gcp-v2-x86
+* tier-5-gcp-v2-x86
+| us-east1
+a| * us-east1-b
+* us-east1-c
+* us-east1-d
+a| * tier-1-gcp-v2-x86
+* tier-2-gcp-v2-x86
+* tier-3-gcp-v2-x86
+* tier-4-gcp-v2-x86
+* tier-5-gcp-v2-x86
+| us-east4
+a| * us-east4-a
+* us-east4-b
+* us-east4-c
+a| * tier-1-gcp-v2-x86
+* tier-2-gcp-v2-x86
+* tier-3-gcp-v2-x86
+* tier-4-gcp-v2-x86
+* tier-5-gcp-v2-x86
+| us-west1
+a| * us-west1-a
+* us-west1-b
+* us-west1-c
+a| * tier-1-gcp-v2-x86
+* tier-2-gcp-v2-x86
+* tier-3-gcp-v2-x86
+* tier-4-gcp-v2-x86
+* tier-5-gcp-v2-x86
+|===
+--
+Amazon Web Services (AWS)::
++
+--
+|=== 
+| Region | Zones | Available throughput tiers
+| af-south-1 
+a| * afs1-az1
+* afs1-az2 
+* afs1-az3
+a| 
```

Comment:
If there are no available throughput tiers for a region / zone, does that mean the region and zone itself is not available for this cluster type?

Reply:
Right, I've removed them!